src/utils/pagedIterator.ts

```diff
@@ -41,13 +42,14 @@ async function nextPageLength<T extends PolkadotCodecType>(
     perPage: number,
     startKey?: StorageKey
 ): Promise<number> {
-    // requesting ENTRIES using the last key as a cut-off point throws a substrate 
+    // requesting ENTRIES using the last key as a cut-off point throws a substrate
     // error however, requesting KEYS with the last key only returns an empty array
     // the latter method can be used to check if all entries were "consumed"
     let keys: StorageKey[] = [];
     const start = startKey ? startKey.toHex() : undefined;
-    keys = await polkadotListings.keysPaged(
-        { pageSize: perPage, startKey: start } as PaginationOptions
-    ) as StorageKey[];
+    keys = (await polkadotListings.keysPaged({
```

Comment:
Why do we need this to be declared beforehand? I think it could be declared here.